paths_cli/tests/compiling/test_cvs.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import yaml
+
+from openpathsampling.tests.test_helpers import data_filename
+import numpy.testing as npt
+
+from paths_cli.compiling.cvs import *
+from paths_cli.compiling.errors import InputError
+import openpathsampling as paths
+from openpathsampling.experimental.storage.collective_variables \
+        import MDTrajFunctionCV
+import mdtraj as md
+
+
+class TestMDTrajFunctionCV:
+    def setup(self):
+        self.ad_pdb = data_filename("ala_small_traj.pdb")
+        self.yml = "\n".join([
+            "name: phi", "type: mdtraj", "topology: " + self.ad_pdb,
+            "period_min: -np.pi", "period_max: np.pi",
+            "func: {func}",
+            "kwargs:", "  {kwargs}",
+        ])
+        self.kwargs = "indices: [[4, 6, 8, 14]]"
+
+    def test_build_mdtraj_function_cv(self):
+        _ = pytest.importorskip('simtk.unit')
```

Comment:
This will start failing in future versions of openmm...

Reply:
Moved all OpenMM import stuff into a separate `paths_cli.compat.openmm`.